src/app/Dashboard/Dashboard.tsx

```diff
@@ -84,7 +90,12 @@ const PlaceholderCard: React.FunctionComponent<
   } & DashboardCardProps
 > = (props) => {
   return (
-    <Card isRounded>
+    <ResizableCard
+      dashboardId={props.dashboardId}
+      minimumSpan={AllPlaceholderCardDescriptor.minimumSpan}
```

Comment:
Probably extract this minimumSpan into a const and use them for all placeholderCardDescriptor?